src/client/java/teammates/client/scripts/helper/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,41 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class that provides methods to help with migrating data to its desanitized form.
+ * Contains a private constructor to prevent instantiation of the class.
```

Comment:
Actually, I was thinking you can omit the constructor altogether (its an empty constructor anyway). Given the class has only static methods, there is no point instantiating it but also there is no point in writing extra code to block instantiation either.